src/main/java/com/tictactoe/app/game/GameRepository.java

```diff
@@ -0,0 +1,68 @@
+package com.tictactoe.app.game;
+
+import com.tictactoe.app.game.configGame.ConfigGame;
+import com.tictactoe.app.game.entity.Game;
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import java.sql.PreparedStatement;
+import java.sql.Statement;
+import java.util.Objects;
+import java.util.Optional;
+
+@RequiredArgsConstructor
+@Repository
+public class GameRepository {
+
+    private final JdbcTemplate jdbcTemplate;
+    private final ConfigGame configGame;
+
+    public Long saveGame(Game game) {
+        String insertPerson = "INSERT INTO game (player1,player2,state,board,playing) VALUES (?,?,?,?,?)";
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+        jdbcTemplate.update(connection -> {
+            PreparedStatement ps = connection.prepareStatement(insertPerson, Statement.RETURN_GENERATED_KEYS);
+            ps.setString(1, game.getPlayer1().toString());
+            ps.setString(2, game.getPlayer2().toString());
+            ps.setString(3, game.getState().getState());
+            ps.setString(4, game.getBoard().toString());
+            ps.setBoolean(5, game.getIsPlaying());
+            return ps;
+        }, keyHolder);
+        return Objects.requireNonNull(keyHolder.getKey()).longValue();
+    }
+
+    public Optional<Game> getGameById(Long gameId) {
+        String getPerson = "SELECT * FROM game WHERE game.id=" + gameId;
+        return jdbcTemplate.query(getPerson, resultSet -> {
+            if (resultSet.next()) {
+                String player1 = resultSet.getString("player1");
+                String player2 = resultSet.getString("player2");
+                String board = resultSet.getString("board");
+                String state = resultSet.getString("state");
+                boolean playing = resultSet.getBoolean("playing");
+                Game gameFromDB = configGame.getGameFromDB(player1, player2, board, state, playing);
```

Comment:
why do you need (player1, player2, board, state, playing) if you alredy had game id and can select game by id